lv_draw/lv_draw_rbasic.c

```diff
@@ -205,9 +216,31 @@ void lv_rletter(const lv_point_t * pos_p, const lv_area_t * mask_p,
         map_p += (width_byte_bpp) - col_byte_cnt;
     }
 
+    // Count the amount of whitespace that is required.
+    if (!isWhitespace) {
+        for(uint8_t i=0; i<letter_h; i++){
+            if(zeroRow[i] == 1 ) trimTop++; else break;
```

Comment:
for readability's sake this would be better like this:
```
if(zeroRow[i] == 1) 
    trimTop++; 
else 
    break;
```

resist your serial killer urges... 🔪